src/Sulu/Bundle/AdminBundle/Resources/js/utils/Ajv/formats/tests/idnEmailValidator.test.js

```diff
@@ -5,10 +5,6 @@ test('Normal email address should pass validation', () => {
     expect(idnEmailValidator('hello@example.com')).toBe(true);
 });
 
-test('Email address with emojis should pass validation', () => {
-    expect(idnEmailValidator('🤌@😎')).toBe(true);
-});
-
```

Comment:
Test was introduced by @luca-rath in https://github.com/sulu/sulu/pull/5815 but even browsers yet does not support this. So I think we can safely remove it:

<img width="300" alt="Bildschirmfoto 2022-11-30 um 01 21 48" src="https://user-images.githubusercontent.com/1698337/204678300-7f79ba76-f42f-4d71-9ef0-e006cc0d0452.png">